WLogger/Model/Log.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Log.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/10/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+import Argo
+import Curry
+
+public struct Log {
```

Comment:
Yeah same bullshit happen in Syrmo. This is what we did

```swift
public static func from(object: PFObject) -> SkateTrick? {
        let partial = createSkateTrick
            <^> object.objectId
            <*> object <| "name"
            <*> Optional.Some(Location.from(object))
        let partial2 = partial
            <*> object.createdAt
            <*> object.updatedAt
            <*> SkateTrickStats.from(object)
        let partial3 = partial2
            <*> getURLFrom(object, key: "trickAnimationFile")
        return partial3
    }
```

You can see the full file [here](https://github.com/syrmo/syrmo-ios/blob/master/Syrmo/Repositories/Parse/PFObject/SkateTrick+PFObject.swift)